pdr_backend/lake/table.py

```diff
@@ -35,7 +36,7 @@ def is_etl_table(table_name: str) -> bool:
 
 
 @enforce_types
-def drop_tables_from_st(pds: PersistentDataStore, type_filter: str, st):
+def drop_tables_from_st(pds: PersistentDataStore, type_filter: str, st: UnixTimeMs):
```

Comment:
this is the only required change. Please revert the unnecessary conversions in cli_module_lake.

Reply:
not sure what mypy will say, but sure

Reply:
done, ty